packages/@sfx/sayt-driver-plugin/test/unit/browser/sayt-driver-plugin.test.ts

```diff
@@ -0,0 +1,193 @@
+import { expect, spy, stub } from '../../utils';
+import { SaytDriverPlugin } from '../../../src/index';
+import { DomEventsPlugin } from '../../../../dom-events-plugin/src/index';
+import { SaytPlugin } from '../../../../sayt-plugin/src';
+
+describe('Sayt Driver Plugin', () => {
+  let Driver: any;
+
+  beforeEach(() => {
+    Driver = new SaytDriverPlugin();
+    Driver.register({
+      'dom_events': new DomEventsPlugin().register({}),
+      'sayt': new SaytPlugin({
```

Comment:
For these two, instead of creating full plugins, it would be better to create stub instances. That way, you don't have to rely on either plugin actually working.

https://sinonjs.org/releases/latest/stubs/#var-stub--sinonstubobj

Alternatively—and this may be the better option—don't bother importing these and simply create objects that have the required methods.